### PR TITLE
[Spotless] Applying Google Code Format for opensearch directory #16

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Applies spotless for opensearch directory in project. Sets checkstyle to not fail on failure for opensearch directory.
 

### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java [hunk 1]
```diff
@@ -35,31 +34,30 @@
...
 public abstract class LuceneQuery {
 
   /**
-   * Check if function expression supported by current Lucene query.
-   * Default behavior is that report supported if:
-   *  1. Left is a reference
-   *  2. Right side is a literal
+   * Check if function expression supported by current Lucene query. Default behavior is that report
+   * supported if:<br>
+   * 1. Left is a reference<br>
```
Comment (Yury-Fridlyand) [inline opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java:43]: Use a list

File: opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/FunctionParameterRepository.java [hunk 2]
```diff
@@ -31,192 +31,254 @@
...
+                  Arrays.stream(SimpleQueryStringFlag.class.getEnumConstants())
+                      .map(Enum::toString)
+                      .collect(Collectors.joining(", "))))
+          .put(
+              "time_zone",
+              "For more information, follow this link: "
+                  + "https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html#of-java.lang.String-")
+          .put(
+              "fuzziness",
+              "Available values are: " + "'AUTO', 'AUTO:x,y' or z, where x, y, z - integer values.")
```
Comment (Yury-Fridlyand) [inline opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/FunctionParameterRepository.java:238]:
```suggestion
              "Available values are: 'AUTO', 'AUTO:x,y' or z, where x, y, z - integer values.")
```

File: opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchAggregationResponseParserTest.java [hunk 3]
```diff
@@ -34,127 +33,116 @@
...
   void no_bucket_one_metric_should_pass() {
-    String response = "{\n"
-        + "  \"max#max\": {\n"
-        + "    \"value\": 40\n"
-        + "  }\n"
-        + "}";
-    NoBucketAggregationParser parser = new NoBucketAggregationParser(
-        new SingleValueParser("max")
-    );
+    String response = "{\n" + "  \"max#max\": {\n" + "    \"value\": 40\n" + "  }\n" + "}";
```
Comment (Yury-Fridlyand) [inline opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchAggregationResponseParserTest.java:39]: Can you keep it multiline?
```suggestion
    String response =
        "{\n"
            + "  \"max#max\": {\n"
            + "    \"value\": 40\n"
            + "  }\n"
            + "}";
```
If not, simplify the string. I think we can omit whitespaces.
```suggestion
    String response = "{ \"max#max\": { \"value\": 40\ } }";
```

File: opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchAggregationResponseParserTest.java [hunk 4]
```diff
@@ -34,127 +33,116 @@
...
-    String response = "{\n"
-        + "  \"date_histogram#date_histogram\": {\n"
-        + "    \"value\": 40\n"
-        + "  }\n"
-        + "}";
-    NoBucketAggregationParser parser = new NoBucketAggregationParser(
-        new SingleValueParser("max")
-    );
+    String response =
+        "{\n" + "  \"date_histogram#date_histogram\": {\n" + "    \"value\": 40\n" + "  }\n" + "}";
```
Comment (Yury-Fridlyand) [inline opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchAggregationResponseParserTest.java:144]:
```suggestion
        "{\n"
            + "  \"date_histogram#date_histogram\": {\n"
            + "    \"value\": 40\n"
            + "  }\n"
            + "}";
```
or
```suggestion
        "{ \"date_histogram#date_histogram\": { \"value\": 40 } }";
```

**Review by Yury-Fridlyand — APPROVED** [review]

File: opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java (hunk 1 above)
Comment (MitchellGale) [reply opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java]: Made list.

**Review by Yury-Fridlyand — APPROVED** [review]

**Comment by Yury-Fridlyand** [comment]
You have compilation errors

**Review by GumpacG — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**